youxia-q2seqware/src/main/java/io/cloudbindle/youxia/q2seqware/RabbitMQListener.java

```diff
@@ -0,0 +1,163 @@
+package io.cloudbindle.youxia.q2seqware;
+
+import io.cloudbindle.youxia.util.Log;
+
+import java.io.IOException;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.ConsumerCancelledException;
+import com.rabbitmq.client.ShutdownSignalException;
+
+public class RabbitMQListener implements QueueListener {
+
+    private String hostName;
+    private String queueName;
+    private boolean autoAck;
+    private boolean durableQueue;
+    private String pathToINIs = "/tmp/inis4seqware/";
```

Comment:
Use something like http://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory%28java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...%29 unless this needs to be the same between invocations, in which case parameterize this in the ~/.youxia/config